main.js

```diff
@@ -1,16 +1,48 @@
 /* eslint-disable no-unused-vars */
-var $choiceList = document.querySelector('.choice-list')
+var Typed = require('typed.js')
+
 var $submitButton = document.querySelector('#submit')
-var $userChoice = document.querySelector('#user-choice')
-var $userName = document.querySelector('#user')
+var $clearButton = document.querySelector('#clear')
+
+var header = new Typed('.header', {
+  strings: ["Let's grab: McDonalds", "Let's grab: Burger King", "Let's grab: Pizza Hut", 'To Be Determined...'],
+  typeSpeed: 50,
+  backSpeed: 50,
+  smartBackSpace: true,
+  showCursor: false
+})
+
+var slogan = new Typed('.slogan', {
+  strings: ["^10000 Can't decide? Let us decide."],
+  typeSpeed: 80,
+  showCursor: false
+})
 
-function renderList() {
-  var $choice = document.createElement('li')
-  $choice.setAttribute('class', 'list')
-  $choice.textContent = $userChoice.value
-  $choiceList.appendChild($choice)
+function renderList(choiceList, choice) {
+  var $userChoice = document.querySelector('#user-choice')
+  var $userName = document.querySelector('#user')
+  if ($userChoice.value !== '' && $userName.value !== '') {
+    choice = document.createElement('li')
+    choice.setAttribute('class', 'list')
+    choice.textContent = $userChoice.value
+    choiceList.appendChild(choice)
+  }
+}
+
+function clearList(choiceList, choice) {
```

Comment:
Why is the parameter named choice if it is a collection of items? Maybe choices is a better name.